sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -280,6 +280,44 @@ case class AlterViewAsCommand(
   }
 }
 
+/**
+ * A command for users to get views in the given database.
+ * If a databaseName is not given, the current database will be used.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW VIEWS [(IN|FROM) database_name] [[LIKE] 'identifier_with_wildcards'];
+ * }}}
+ */
+case class ShowViewsCommand(
+    databaseName: Option[String],
+    tableIdentifierPattern: Option[String]) extends RunnableCommand {
+
+  // The result of SHOW TABLES/SHOW TABLE has three basic columns: database, tableName and
+  // isTemporary. If `isExtended` is true, append column `information` to the output columns.
```

Comment:
It seems you wrongly copied&pasted this comment from `ShowTablesCommand`.

Reply:
Oops, my mistake. Updated in dc6e3cb21e5de2a2c591b42014cc03fc4418cca3.